components/datadog/agent/docker_image.go

```diff
@@ -17,7 +17,7 @@ const (
 	defaultOTAgentImageTag       = "nightly-ot-beta-main"
 )
 
-func dockerAgentFullImagePath(e config.Env, repositoryPath, imageTag string, otel bool) string {
+func dockerAgentFullImagePath(e config.Env, repositoryPath, imageTag string, otel, jmx bool) string {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Looks like this change is never used, as you change the `FullImagePath` instead of calling this function. Is it intentional?

Reply:
Let me revert it just to change I need for using in the Docker Agent test I was writing.